### PR TITLE
Toggle-Menu

### PR DESCRIPTION
**Pull request by Abrar052**
### Created Responsive and Dynamic Toggle Button

- Updated navbar in index.html
- Added style to the toggle button
- Created index.js to add dynamics to the toggle button
- Updated linters.yml to check linters errors in index.js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,15 +10,23 @@
...
+                    <h2 class="brand"><a href="#section-1">Welcome</a></h2>
+                    <button class="navToggle" id="navToggle"><i class="fas fa-bars"></i></button>
+                </div>
+                <nav>
+                    <ul class="nav-bar" id="nav-desktop">
+                        <li class="nav-desktop" id="navPortfolio"><a href="#projectTitle" class="nav-link">Portfolio</a></li>
+                        <li class="nav-desktop" id="navAbout"><a href="#about" class="nav-link">About</a></li>
+                        <li class="nav-desktop" id="navContact"><a href="#contact" class="nav-link">Contact</a></li>
+                      </ul>
+                </nav>
```
Comment (Nemwel-Boniface) [inline index.html:25]: You have done a great job with your mobile portfolio menu. However, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). You are currently left with the following styling changes to make your project perfect and ready for approval:

- [ ] When you toggle to show the menu, the bars are supposed to be changed with an "X" according to the Figma design. Kindly fix this.

**This is how your design currently looks like:**

image

**This is the desired Figma design:**

image

- [ ] The "Welcome" text is supposed to be hidden according to the chosen Figma design when the menu is opened. Kindly hide it. You can hide it with JavaScript when the menu is activated with something like ```logo.style.display = 'none';```
- [ ] I have also realized that your menu links are not positioned at the correct side of the screen as shown in the Figma design. You can use something like ```text-align: left;``` to align your nav items to the left side of the screen as shown in the Figma design.

**This is how your Figma design currently looks like:**

image

**This is the desired Figma design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Abrar052            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with styling and creating your mobile menu good job :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -10,15 +10,23 @@
...
+            </div>
```
Comment (codecaiine) [inline index.html]: - [ ] [REQUIRED] Good Job so far for implementing some changes :+1:  But you still need to work on your menu As the previous reviewers mentioned, your menu needs to stick to the Figma template as much as possible :smile: 

> You have done a great job with your mobile portfolio menu. However, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). You are currently left with the following styling changes to make your project perfect and ready for approval:
> 
> * [ ]  When you toggle to show the menu, the bars are supposed to be changed with an "X" according to the Figma design. Kindly fix this.
> 
> **This is how your design currently looks like:**
> 
> image
> 
> **This is the desired Figma design:**
> 
> image
> 
> * [ ]  The "Welcome" text is supposed to be hidden according to the chosen Figma design when the menu is opened. Kindly hide it. You can hide it with JavaScript when the menu is activated with something like `logo.style.display = 'none';`
> * [ ]  I have also realized that your menu links are not positioned at the correct side of the screen as shown in the Figma design. You can use something like `text-align: left;` to align your nav items to the left side of the screen as shown in the Figma design.
> 
> **This is how your Figma design currently looks like:**
> 
> image
> 
> **This is the desired Figma design:**
> 
> image


### Your menu now
Screenshot from 2022-07-26 21-29-00

###Figma template
Screenshot from 2022-07-26 21-29-20

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Abrar052 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Great job setting up the correct linters :heavy_check_mark: 
Correct workflow :heavy_check_mark: 
Meaningful commit messages :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @Abrar052 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good PR description :heavy_check_mark: 
- Good project implementation :heavy_check_mark: 
- Nice commit history :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**